topic2/src/main/java/cartService/CartServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package cartService;
+import java.util.LinkedList;
+
+public class CartServiceImpl implements CartService{
+	
+	private LinkedList<Item> items = new LinkedList<Item>();
```

Comment:
What is the difference on doing the creation in here or in a constructor?

Reply:
I can't find a concrete answer to this question, I'm finding mixed responses in stackoverflow

Ej https://stackoverflow.com/questions/4916735/default-constructor-vs-inline-field-initialization

Reply:
Should I change it? Is there a 'best practice' for this??